sqldelight-compiler/src/main/kotlin/timber/log/Timber.kt

```diff
@@ -0,0 +1,326 @@
+package timber.log
```

Comment:
Will remove once `Timber` exposes a jvm artifact

Reply:
Or even better, be multiplatfom. 👀